src/Umbraco.Web.Common/Runtime/AspNetCoreComposer.cs

```diff
@@ -72,7 +72,6 @@ public override void Compose(Composition composition)
             composition.RegisterUnique<IMacroRenderer, MacroRenderer>();
             composition.RegisterUnique<IMemberUserKeyProvider, MemberUserKeyProvider>();
 
-            composition.RegisterUnique<AngularJsonMediaTypeFormatter>();
```

Comment:
JsonSerializerSettings isn't registered in container so it can't ever be resolved, at present the container will build as validation is currently off however that wasn't true at the time this line was removed.

AngularJsonMediaTypeFormatter is never resolved from container at present, it's always manually constructed.

Reply:
>Because it uses Current.SecuritySettings in UserInvite.cs

SHOCKING! 😛

Reply:
Not for long!